src/shadertools/modules/fp64/fp64-arithmetic.spec.js

```diff
@@ -0,0 +1,243 @@
+// Copyright (c) 2015 - 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the 'Software'), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+// Actual tests for different arithmetic functions
+
+/* eslint-disable camelcase, prefer-template, max-len */
+/* global window, document, */
+import test from 'tape-catch';
+import {Buffer, Program, assembleShaders, registerShaderModules, fp64} from 'luma.gl';
+import {initializeGL, initializeTexTarget, render, getGPUOutput} from '../../test/gpu-test-utils';
+
+const BUFFER_DATA = new Float32Array([1, 1, -1, 1, 1, -1, -1, -1]);
+
+// Special utility functions for df64 tests
+
+function fp64ify(a) {
+  const hi = Math.fround(a);
+  const lo = a - Math.fround(a);
+  return new Float32Array([hi, lo]);
+}
+
+function getFloat64(upper = 256) {
+  return Math.random() * Math.pow(2.0, (Math.random() - 0.5) * upper);
+}
+
+export function getRelativeError64(result, reference) {
+  const reference64 = reference[0] + reference[1];
+  const result64 = result[0] + result[1];
+  return Math.abs((reference64 - result64) / reference64);
+}
+
+export function getRelativeError(result, reference) {
+  return Math.abs((reference - result) / reference);
+}
+
+/*
```

Comment:
Nit: remove disabled code or add a comment for any remaining work.
Same for other commented out code below.